### PR TITLE
Add back 2.5.0 to the compatibility tests

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
2.5.1 introduces a new verison of the trigger library.

### CODE REVIEW

File: compatibility/versions.bzl [hunk 1]
```diff
@@ -27,7 +27,7 @@ sdk_versions = [
     "2.2.1",
     "2.3.8",
     "2.4.2",
-    "2.5.1",
```
Comment (cocreature) [inline compatibility/versions.bzl:30]: this file is autogenerated from `UpdateVersion.hs`. I think if you want to catch this, then update the script to include both 2.5.0 and 2.5.1 in there.

Comment (remyhaemmerle-da) [reply compatibility/versions.bzl:30]: Good catch.  I was not aware of this Haskell file.

File: compatibility/versions/UpdateVersions.hs [hunk 2]
```diff
@@ -35,7 +35,7 @@ instance Semigroup Versions where
     Versions a <> Versions b = Versions (a <> b)
 
 minimumVersion :: Version
-minimumVersion = SemVer.incrementMajor SemVer.initial
+minimumVersion = SemVer.version 2 3 0 [] []
```
Comment (garyverhaegen-da) [inline compatibility/versions/UpdateVersions.hs:38]: That seems unrelated to the issue of including 2.5.0 - would you mind commenting on the change here? Per [the documentation](https://hackage.haskell.org/package/semver-0.4.0.1/docs/Data-SemVer.html) it looks like this previously evaluated to `1.0.0`, so switching to `2.3.0` seems like a big change.

**Review by garyverhaegen-da — APPROVED** [review]
